app/src/app.js

```diff
@@ -0,0 +1 @@
+import Hello from './component/hello.jsx';
```

Comment:
Please create in components and containers folders file with name index.js with all exported components and containers

export Hellow from './Hellow/Hellow';

and use in  app.js as import {Hellow} from './components' or './containers'